src/platform/forms/save-in-progress/ApplicationStatus.jsx

```diff
@@ -67,17 +67,18 @@ export class ApplicationStatus extends React.Component {
       applyLink,
       formConfig,
     } = this.props;
+    const appType = formConfig?.customText?.appType ?? APP_TYPE_DEFAULT;
+    const continueAppButtonText =
+      formConfig?.customText?.continueAppButtonText ??
+      CONTINUE_APP_DEFAULT_MESSAGE;
+    const startNewAppButtonText =
+      formConfig?.customText?.startNewAppButtonText ??
+      START_NEW_APP_DEFAULT_MESSAGE;
+    const appAction = formConfig?.customText?.appAction ?? APP_ACTION_DEFAULT;
```

Comment:
If we're going to change this, I would go with my previous suggestion.

```suggestion
    const {
      startNewAppButtonText = START_NEW_APP_DEFAULT_MESSAGE,
      continueAppButtonText = CONTINUE_APP_DEFAULT_MESSAGE,
      appType = CONTINUE_APP_DEFAULT_MESSAGE,
      appAction = APP_ACTION_DEFAULT,
    } = this.props.routes[1].formConfig?.customText || {};
```